rebasehelper/cli.py

```diff
@@ -207,6 +213,13 @@ def build_parser():
                  "and note that = before it is mandatory"
         )
         parser.add_argument(
+            "--srpm-options",
```

Comment:
Definitely use `--srpm-builder-options`. The alternative is shorter but confusing.